### ISSUE TITLE
Can't open alternate mailbox

### ISSUE THREAD

**Issue by phw198** [phw198/OutlookGoogleCalendarSync#175]
<div><strong>This discussion was imported from <a href="http://outlookgooglecalendarsync.codeplex.com/discussions/659613" target="_blank">CodePlex</a></strong></div><hr/>
<div>
<p><strong><a href="http://www.codeplex.com/site/users/view/StarNamer" target="_blank">StarNamer</a></strong> wrote at 2016-11-22 09:51:</p><div class="markDownOutput ">I have an Outlook profile which opens 2 addtional mailbox, but despite the comment
<a href="https://outlookgooglecalendarsync.codeplex.com/discussions/639136"  rel="nofollow">
here</a> the alternate mailbox option is still greyed out and I can't open it. Does this really mean alternate profile?<br>
</div></div>
<hr/>
<div>
<p><strong><a href="http://www.codeplex.com/site/users/view/StarNamer" target="_blank">StarNamer</a></strong> wrote at 2016-11-22 09:54:</p><div class="markDownOutput ">No, it's not profiles. Tried adding a second account with the email address of one of the secondary mailboxes, but it didn't find anything (and there is at least one all-day event every day!). Is it still using my default calendar or is there some extra
 setting I need to access a shared mailbox's calendar?<br>
<br>
To summarise what I'm trying to do. In additional to my personal calendar, we have a team calendar in another mailbox which all our Outlook accounts also open. I'd like to be able to merge entries from that to my Google account as well. Ideally, I'd prefer
 not to add it as a second mailbox as it gets a lot of email, most of which I'm not interested in. I just need to access it's calendar. Note that most of the entries I'm interested in are All-Day Events (if that makes a difference).<br>
</div></div>

**Comment by phw198** [comment]
Think you've answered your own question! `Alternate mailbox` is not another profile, it's another mailbox - such as the one you don't want to add as a second (alternate) mailbox because "it gets lots of email".

In v2.4 there is now the option to sync a shared Outlook calendar, so that might provide another option for you.
